lib/seattleflu/db/cli/command/etl/clinical.py

```diff
@@ -224,28 +225,41 @@ def encounter_details(document: dict) -> dict:
         }
 
 
-def race(race_name: str) -> list:
+def race(races: str or list) -> list:
     """
-    Given a *race_name*, returns the matching race identifier found in Audere
-    survey data.
+    Given one or more *races*, returns the matching race identifier found in
+    Audere survey data.
     """
-    if race_name is None:
+    if races is None:
         LOG.debug("No race response found.")
         return [None]
 
+    if not isinstance(races, list):
+        races = [races]
+
     race_map = {
         "American Indian or Alaska Native": "americanIndianOrAlaskaNative",
+        "amerind": "americanIndianOrAlaskaNative",
         "Asian": "asian",
         "Black or African American": "blackOrAfricanAmerican",
+        "black": "blackOrAfricanAmerican",
```

Comment:
This line is part of _etl clinical: edit `race` to take str or list_ but seems like it should be part of _etl clinical: add mapping for KP variables_.